docs/src/index.js

```diff
@@ -5,8 +5,9 @@ import App from './components/App';
 import CardPage from './components/CardPage';
 import { render } from 'react-dom';
 import { getCards } from 'corkboard/init';
+import '!style-loader!css-loader!gestalt/dist/gestalt.css';
```

Comment:
don't feel strongly either way, but i think we could also just add `noParse: /gestalt/` to the webpack config

Reply:
My next step is to just have the docs use CRA so no webpack at all! Didn’t want to spent too much time tidying this up so just followed conventions.

Reply:
My next step is to just have the docs use CRA so no webpack at all! Didn’t want to spent too much time tidying this up so just followed conventions.